### ISSUE TITLE
How to debug `interact` giving text only, no widget

### ISSUE THREAD

**Issue by denis-bz** [jupyter-widgets/ipywidgets#3025]
The [Ipywidgets FAQ](https://ipywidgets.readthedocs.io/en/stable/user_install.html#frequently-asked-questions) says "When  display a widget or interact, I just see some text ..."
I'm getting this too; how could I debug it ?
1) which tests in `widgets/tests/test_interaction.py` could I run ?
2) set breakpoints in `widgets/interaction.py` ?
3) test node.js standalone ?

(I've also asked on SO: [how-to-install-ipywidgets-and-run-interact-in-ipython](https://stackoverflow.com/questions/62306962/how-to-install-ipywidgets-and-run-interact-in-ipython) )

My versions: macos 10.10.5  python 3.7.6  ipython-7.19.0  ipywidgets-7.5.1  jupyter_core-4.7.0  matplotlib-3.3.3  node v12.18.0
no virtualenv

Thanks, cheers

**Comment by vidartf** [comment]
Are you an jupyterlab or jupyter notebook? And which versions?

**Comment by denis-bz** [comment]
@vidartf, neither, I want iteractive widgets in IPython. If that doesn't work, could you say so in the doc ?

Bytheway https://mpl-interactions.readthedocs.io does work in IPython (with ugly mpl sliders).
Thanks, cheers

**Comment by ianhi** [comment]
@denis-bz I don't think that they will work in ipython - to me that seems like a reasonable addition to the docs. Where would it have helped you to have a sentence about that?

>  does work in IPython (with ugly mpl sliders).

This inspired me to open https://github.com/matplotlib/matplotlib/issues/19256. Which even if it gets shot down there I will probably incorporate into mpl-interactions. So you're causing real changes here :)
